index.test.js

```diff
@@ -6,23 +6,47 @@ const { relative, resolve } = require('path')
 const sinon = require('sinon')
 
 const testContext = { name: 'Test Name' }
-const testRecipients = ['test@example.com']
-const testSender = 'sender@example.com'
-const testSubject = `${testContext.name}, Test Subject`
+
+const testHtmlTemplate =
+  '<!DOCTYPE html>\n' +
+  '<html lang="en">\n' +
+  '  <head></head>\n' +
+  '  <body>\n' +
+  '    <p>Name: {{ name }}</p>\n' +
+  '  </body>\n' +
+  '</html>\n'
+
+const testTextTemplate = 'This is a test text message to {{ name }}'
+
 const testHtml =
   '<!DOCTYPE html>\n' +
   '<html lang="en">\n' +
   '  <head></head>\n' +
   '  <body>\n' +
-  '    <p>Name: test-context</p>\n' +
+  '    <p>Name: Test Name</p>\n' +
   '  </body>\n' +
   '</html>\n'
 
 const testMessage = {
-  to: testRecipients,
-  from: testSender,
-  subject: testSubject,
-  html: testHtml
+  to: 'to@ignoreme.com',
+  from: 'from@ignoreme.com',
+  replyTo: 'reply@ignoreme.com',
+  subject: 'This is a plain text subject',
+  html: testHtml,
+  text: 'This is a plain text email message.',
+  cc: 'cc@ignoreme.com',
+  bcc: 'bcc@ignoreme.com'
+}
+
+const responseWhenTemplatesPresent = {
+  to: 'to@ignoreme.com',
+  from: 'from@ignoreme.com',
+  replyTo: 'reply@ignoreme.com',
+  subject: 'This is a plain text subject',
+  html: testHtml,
+  text: 'This is a test text message to Test Name',
+  cc: 'cc@ignoreme.com',
```

Comment:
here and the `testMessage` above, bump these two (cc and bcc) up to be under to (so that it follows standard email definitions

Reply:
done